types/office-js/index.d.ts

```diff
@@ -279,9 +279,25 @@ declare namespace Office {
      * Specifies the result of an asynchronous call.
      * 
      * @remarks
-     * <table><tr><td>Hosts</td><td>Access, Excel, Outlook, PowerPoint, Project, Word</td></tr></table>
      * 
      * Returned by the status property of the AsyncResult object.
+     * 
+     * **Support details**
+     * 
+     * A capital Y in the following matrix indicates that this enumeration is supported in the corresponding Office host application. An empty cell indicates that the Office host application doesn't support this enumeration.
+     * 
+     * For more information about Office host application and server requirements, see [Requirements for running Office Add-ins](https://docs.microsoft.com/office/dev/add-ins/concepts/requirements-for-running-office-add-ins).
```

Comment:
need to update the link notation, right?

Reply:
Oh you're totally right. Good catch.